### DISCUSSION TITLE
Hi Tsyuay, can you add farm bee game please??

### DISCUSSION THREAD

**Discussion by tnt3333334** [Tsyuay/FrivReborn#33] [Q&A] [answered]
<img width="921" alt="Sem título" src="https://user-images.githubusercontent.com/122381404/212089600-ef368457-73ab-4d0b-9855-52bfa520418d.png">

Please!!

**Comment by tnt3333334** [comment]
I also wanted you to add this game that was my brother's favorite.


<img width="960" alt="Sem título" src="https://user-images.githubusercontent.com/122381404/212095483-5cb40378-a02f-4b22-a983-21f7e8b54dbe.png">

**Comment by tnt3333334** [comment]
do it please


<img width="796" alt="Sem título" src="https://user-images.githubusercontent.com/122381404/212355290-2b7f5c97-e61e-4460-a171-53b7561559f2.png">

**Comment by Tsyuay** [comment, marked as answer]
done
